src/pages/Proposal/components/ProposalSearch/index.jsx

```diff
@@ -44,7 +45,7 @@ const ProposalSearch = ({ selectMehtod = 'ReleaseApprovedContract' }) => {
     const wrapperHeight = popup.clientHeight;
     const innerHeight = popupChild?.clientHeight;
     const toBottom = innerHeight - wrapperHeight;
-    if ((popup.scrollTop + toBottomDistance > toBottom) && !isFetch) {
+    if (popup.scrollTop + toBottomDistance > toBottom && !isFetch) {
```

Comment:
Code Complete：Chapter 19 'Pease use parentheses to make boolean expressions clearer.'

Reply:
Parentheses deleted by eslint formatOnSave. I will add it manually.